dactyl-config.yml

```
@@ -4427,4 +4420,4 @@ known_broken_links:
 
 word_substitutions_file: tool/word_substitutions.yaml
 phrase_substitutions_file: tool/phrase_substitutions.yaml
-spelling_file: tool/spelling.txt
+spelling_file: tool/spelling.txt
```

Please leave the newline on the end of the file here.

```suggestion
spelling_file: tool/spelling.txt

```